src/views/templates/details/TooltipNoEditPermissions.tsx

```diff
@@ -1,29 +0,0 @@
-import React, { FC } from 'react';
-
-import { Tooltip } from '@patternfly/react-core';
-
-import { NO_EDIT_TEMPLATE_PERMISSIONS } from '../utils/constants';
-
-import './tooltip-no-edit-permissions.scss';
-
-type TooltipNoEditPermissionsProps = {
-  hasEditPermission: boolean;
-};
-
-const TooltipNoEditPermissions: FC<TooltipNoEditPermissionsProps> = ({
-  hasEditPermission,
-  children,
-}) => {
-  if (!hasEditPermission) {
-    return (
-      <Tooltip content={NO_EDIT_TEMPLATE_PERMISSIONS}>
-        {/* Span here as disabled buttons do not fire any event. We need the 'hover' event. */}
-        <span>{children}</span>
-      </Tooltip>
-    );
-  }
-
-  return <>{children}</>;
-};
-
-export default TooltipNoEditPermissions;
```

Comment:
I think this component is not bad, and can be expanded for other cases like VM, MigrationPolicies, DataSources etc. adding a prop for the tooltip content pretty much settels it, or am I missing something

Reply:
yes. I've just noticed that this component is used in combination with a disabled button so i created `ButtonWithPermissionTooltip](https://github.com/kubevirt-ui/kubevirt-plugin/pull/970/files#diff-fbc8d130302d440edfcbeac587df7101fb481eedc25efd0504f8faaef3cc7082)`